xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -1475,6 +1475,7 @@ core.register.welcome=Sign up here so you can edit pages and participate in the
 core.register.passwordMismatch=Passwords are different or password is empty.
 core.register.userAlreadyExists=User already exists.
 core.register.invalidUsername=Invalid username provided. Please use only letters from the latin alphabet, numbers, and the underscore character '_'.
+core.register.mailSenderWronglyConfigured=The user has been created but the validation email has not been sent. Please check the Mail Sending Configuration and consider to recreate the user.
```

Comment:
"Please check the Mail Sending Configuration and consider to recreate the user." --> "Please check the Mail Sending Configuration and consider recreating the user."